flink-python/pyflink/table/table_config.py

```diff
@@ -38,24 +44,76 @@ def is_stream(self, is_stream):
 
     @property
     def parallelism(self):
+        """
+        The parallelism for all operations.
+        """
         return self._parallelism
 
     @parallelism.setter
     def parallelism(self, parallelism):
         self._parallelism = parallelism
 
+    @property
+    def timezone(self):
+        """
+        The timezone_id for a timezone, either an abbreviation such as "PST", a full name such as
```

Comment:
`The timezone_id for a timezone` ->  `Returns the timezone id`

Reply:
Done.